homeassistant/components/accuweather/__init__.py

```diff
@@ -107,6 +117,10 @@ def __init__(self, hass, session, api_key, location_key, forecast: bool):
         update_interval = (
             timedelta(minutes=64) if self.forecast else timedelta(minutes=32)
         )
+        # If less frequent updates option is enabled data updates will be performed
+        # twice as rarely.
+        if less_frequent_updates:
+            update_interval *= 2
```

Comment:
I think we should not introduce these kind of options. They are confusing and actually do not add that much.

Instead, we should have sane defaults for our update intervals. For weather integration, a default of once an hour is fine by default. We don't need a option for that.

Reply:
So I close this PR and open a new one with the change of `update_interval`.